requirements.txt

```diff
@@ -1,7 +1,7 @@
-absl-py==1.4.0
+absl-py==2.1.0
 numpy==1.24.4
-ortools==9.6.2534
-protobuf==4.23.3
+ortools==9.9.3963
+protobuf==5.26.1
```

Comment:
Why is protobuf required ? Doesn't pip resolve dependencies itself ?

Reply:
Yes this true but it was already there so I just updated the version number, I can try to remove it, as well as absl

Reply:
I confirm that pip resolves the dependency just file
```
pip install ortools==9.9.3963
Downloading absl_py-2.1.0-py3-none-any.whl (133 kB)
   ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 133.7/133.7 kB 32.6 MB/s eta 0:00:00
Downloading immutabledict-4.2.0-py3-none-any.whl (4.7 kB)
Downloading pandas-2.2.2-cp311-cp311-manylinux_2_17_x86_64.manylinux2014_x86_64.whl (13.0 MB)
   ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 13.0/13.0 MB 26.1 MB/s eta 0:00:00
Downloading protobuf-5.26.1-cp37-abi3-manylinux2014_x86_64.whl (302 kB)
```